oam/install_scripts/columnstore-pre-uninstall.in

```diff
@@ -82,7 +82,7 @@ if [ -n "$systemctl" ] && [ $(running_systemd) -eq 0 ]; then
     systemctl disable mcs-dmlproc > /dev/null 2>&1
     systemctl disable mcs-exemgr > /dev/null 2>&1
     systemctl disable mcs-primproc > /dev/null 2>&1
-    systemctl disable mcs-workernode > /dev/null 2>&1
+    systemctl disable mcs-workernode@.service > /dev/null 2>&1
```

Comment:
This service suffix caught me b/c I asked if it means something. I believe we should make all these disable calls the same thus should be fine w/o .service.